### PR TITLE
releasing mistral-7b-instruct-v02

### PR DESCRIPTION
**Pull request by choudharykartik**
model_link:  [mistralai-Mistral-7B-Instruct-v0-2:3 - Azure AI | Machine Learning Studio](https://ml.azure.com/model/mistralai-Mistral-7B-Instruct-v0-2:3/details?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-kartik2&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#overview) 
endpoint link: [[mistralai-mistral-7b-instruct-3 - Azure AI | Machine Learning Studio](https://ml.azure.com/endpoints/realtime/mistralai-mistral-7b-instruct-3/test?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-kartik2&tid=72f988bf-86f1-41af-91ab-2d7cd011db47)](https://ml.azure.com/endpoints/realtime/mistralai-mistral-7b-instruct-4/detail?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-kartik2&tid=72f988bf-86f1-41af-91ab-2d7cd011db47)
sample response:
image

save to blob job link: https://ml.azure.com/runs/serene_fish_11w9vkkyrq?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourcegroups/shared-finetuning-rg/workspaces/v-kartik2&tid=72f988bf-86f1-41af-91ab-2d7cd011db47

MLModel
```
flavors:
  hftransformersv2:
    code: null
    config_hf_load_kwargs:
      trust_remote_code: true
    hf_config_class: AutoConfig
    hf_pretrained_class: AutoModelForCausalLM
    hf_tokenizer_class: AutoTokenizer
    huggingface_id: mistralai/Mistral-7B-Instruct-v0.2
    model_data: data
    model_hf_load_args:
      trust_remote_code: true
    pytorch_version: 1.13.1
    task_type: chat-completion
    tokenizer_hf_load_kwargs:
      trust_remote_code: true
    transformers_version: 4.38.2
  python_function:
    data: data
    env: conda.yaml
    loader_module: azureml.evaluate.mlflow.hftransformers
    python_version: 3.8.18
metadata:
  base_model_name: mistralai/Mistral-7B-Instruct-v0.2
  azureml.base_image: mcr.microsoft.com/azureml/curated/foundation-model-inference:31
  base_model_task: chat-completion
  is_acft_model: true
  is_finetuned_model: false
mlflow_version: 2.10.2
model_uuid: a2e7f441fd904bfe93604d6219d99899
utc_time_created: '2024-04-02 10:02:58.717060'
```

conda.yaml
```
channels:
- conda-forge
dependencies:
- python=3.8.18
- pip<=23.3.1
- pip:
  - mlflow==2.10.2
  - cloudpickle==2.2.1
  - jsonpickle==3.0.3
  - mlflow-skinny==2.10.2
  - azureml-core==1.55.0.post2
  - azureml-mlflow==1.55.0
  - azureml-metrics[all]==0.0.46.post1
  - scikit-learn==1.1.3
  - cryptography==42.0.5
  - python-dateutil==2.9.0.post0
  - datasets==2.18.0
  - soundfile==0.12.1
  - librosa==0.10.1
  - diffusers==0.27.2
  - sentencepiece==0.2.0
  - transformers==4.38.2
  - torch==1.13.1
  - accelerate==0.28.0
  - Pillow==10.2.0
  - azureml-evaluate-mlflow==0.0.46.post1
- pycocotools=2.0.4
name: mlflow-env
```

### CODE REVIEW

File: assets/models/system/Mistral-7B-Instruct-v0-2/description.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+For full details of this model please read our [paper](https://arxiv.org/abs/2310.06825) and [release blog post](https://mistral.ai/news/la-plateforme/).
+
+## Model Architecture
+
+This instruction model is based on Mistral-7B-v0.1, a transformer model with the following architecture choices:
+- Grouped-Query Attention
+- Sliding-Window Attention
+- Byte-fallback BPE tokenizer
+
+## Limitations
```
Comment (novaturient95) [inline assets/models/system/Mistral-7B-Instruct-v0-2/description.md:18]: Please refactor description to match sections described in [document](https://microsoftapc-my.sharepoint.com/:w:/g/personal/ayushmishra_microsoft_com/ETjbEy6poUhBsdysFKduH3IBmrdEe2gP2Yzza3p6COAAhA?e=mKdodm)

File: assets/models/system/Mistral-7B-Instruct-v0-2/description.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+The Mistral 7B Instruct model is a quick demonstration that the base model can be easily fine-tuned to achieve compelling performance. 
+It does not have any moderation mechanisms. We're looking forward to engaging with the community on ways to
+make the model finely respect guardrails, allowing for deployment in environments requiring moderated outputs.
+
+# **Inference samples**
+
```
Comment (novaturient95) [inline assets/models/system/Mistral-7B-Instruct-v0-2/description.md:25]: add inference samples

File: assets/models/system/Mistral-7B-Instruct-v0-2/description.md (hunk 1 above)
Comment (novaturient95) [reply assets/models/system/Mistral-7B-Instruct-v0-2/description.md:18]: Please take reference from bert-base-uncased model how description should be formatted

File: assets/models/system/Mistral-7B-Instruct-v0-2/spec.yaml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+      Standard_ND40rs_v2,
+      Standard_NC24ads_A100_v4,
+      Standard_NC48ads_A100_v4,
+      Standard_NC96ads_A100_v4,
+      Standard_ND96amsr_A100_v4,
+      Standard_ND96asr_v4,
+    ]
+  inference_supported_envs:
+    - vllm
+    - ds_mii
```
Comment (novaturient95) [inline assets/models/system/Mistral-7B-Instruct-v0-2/spec.yaml:28]: was ds_mii tested? If not please remove it from the list

File: assets/models/system/Mistral-7B-Instruct-v0-2/description.md (hunk 1 above)
Comment (choudharykartik) [reply assets/models/system/Mistral-7B-Instruct-v0-2/description.md]: Updated

File: assets/models/system/Mistral-7B-Instruct-v0-2/description.md (hunk 2 above)
Comment (choudharykartik) [reply assets/models/system/Mistral-7B-Instruct-v0-2/description.md:23]: Added

File: assets/models/system/Mistral-7B-Instruct-v0-2/spec.yaml (hunk 3 above)
Comment (choudharykartik) [reply assets/models/system/Mistral-7B-Instruct-v0-2/spec.yaml]: Removed

**Review by novaturient95 — APPROVED** [review]

**Pull request merged**